nucleus/distributions/pom.xml

```diff
@@ -148,6 +148,7 @@
                         <attach>false</attach>
                         <appendAssemblyId>false</appendAssemblyId>
                         <useProjectArtifact>false</useProjectArtifact>
+                        <ignoreMissingDescriptor>true</ignoreMissingDescriptor>
```

Comment:
The same property is set to false here. Also I believe false is correct - if the descriptor is missing, the build is broken.